lan867x/examples/lan867x_tcp_sockets/client/main/client.c

```diff
@@ -49,8 +51,21 @@ void app_main(void)
     ESP_ERROR_CHECK(esp_netif_attach(eth_netif, esp_eth_new_netif_glue(eth_handles[0])));
     // Register user defined event handers
     ESP_ERROR_CHECK(esp_event_handler_register(IP_EVENT, IP_EVENT_ETH_GOT_IP, &got_ip_event_handler, NULL));
+    // Configure PLCA
+#if CONFIG_EXAMPLE_LAN867X_USE_PLCA
+    // Configure PLCA with node number from config
+    uint8_t plca_id = CONFIG_EXAMPLE_LAN867X_PLCA_ID;
+    esp_eth_ioctl(eth_handles[0], LAN867X_ETH_CMD_S_PLCA_ID, &plca_id);
+    bool plca_en = false;
```

Comment:
Does it mean PLCA is disable? I also don't see any other settings of PLCA. Does it mean it works in some default configuration? If so, we should describe the default in README.

Reply:
It's a leftover from me testing CSMA/CD. Changed back to `true`.

Reply:
As there is no point in configuring additional transmit opportunities or timings only the bare minimum is being set here, i.e. the ID and enable bit.

Reply:
I think that at least "NCNT[7:0] Node Count" should be configurable for coordinator to ensure at least some basic optimization of the bus.

Reply:
Node count is configured in `server/main/server.c`. Nodes configured with ID other than 0 ignore this field.

Reply:
I see. Thanks.